docker-compose.yaml

```diff
@@ -20,5 +20,19 @@ services:
       # certificates
       CERT_VERSION: '1'
       CERT_PATH: './pkg/storage/certificates'
+    ports: 
+      - "8081:8080"
+
+  http:
+    container_name: "http"
+    build: 
+      context: "./"
+      dockerfile: "./docker/http.Dockerfile"
+    environment:
+      HTTP_ADDRESS: ':8080'
+      JWT_ADDRESS: 'auth:8081'
+      GRPC_ADDRESS: ':50051'
```

Comment:
`GRPC_ADDRESS` still need to update to something `grpc:50051`. But this should be done when grpc cmd is available.

Reply:
Now it is empty so I will change it when service will be ready)